configure.ac

```
@@ -729,25 +739,35 @@ AS_IF([test x"$enable_str_lib" != "xno"],
 ## TODO: have two values, one for host and one for target
 AC_SYS_INTERPRETER
 
-long_shebang=false
-AS_IF(
-  [test "x$interpval" = "xyes"],
-    [AS_CASE([$host],
-      [*-cygwin|*-*-mingw32*|*-pc-windows],
-        [shebangscripts=false],
-      [shebangscripts=true
-       prev_exec_prefix="$exec_prefix"
-       AS_IF([test "x$exec_prefix" = "xNONE"],[exec_prefix="$prefix"])
-       eval "expanded_bindir=\"$bindir\""
-       exec_prefix="$prev_exec_prefix"
-       # Assume maximum shebang is 128 chars; less #!, /ocamlrun, an optional
-       # 1 char suffix and the \0 leaving 115 characters
-       AS_IF([test "${#expanded_bindir}" -gt 115],[long_shebang=true])
-      ]
-    )],
-  [shebangscripts=false]
+shebangscripts=false
+launch_method='exe'
+AS_IF([test "x$interpval" = "xyes"],
+  [AS_CASE([$host],
+    [*-cygwin],
+    [# Cygwin supports shebangs, which we use for the compiler itself, but
+    # partially for legacy, and partially so that executables can be easily run
+    # from outside Cygwin, the executables ocamlc and ocamlopt produce do not
+    # use shebang scripts
+    shebangscripts=true
+    launch_method='sh'
+    AS_IF([test x"$target_launch_method" = 'x'], [target_launch_method='exe'])],
```

Small pet peeve, could you consider indenting this case?
```suggestion
      [# Cygwin supports shebangs, which we use for the compiler itself, but
      # partially for legacy, and partially so that executables can be easily run
      # from outside Cygwin, the executables ocamlc and ocamlopt produce do not
      # use shebang scripts
      shebangscripts=true
      launch_method='sh'
      AS_IF([test x"$target_launch_method" = 'x'], [target_launch_method='exe'])],
```

---

Oh my, not just a pet peeve - I think something must have got altered and then not re-indented!